dev/README.md

```diff
@@ -113,7 +95,7 @@ Would you like to update the associated JIRA? (y/n): y
 Enter comma-separated fix version(s) [11.0.0]:
 ```
 
-```
+```text
```

Comment:
This block seems duplicated

Reply:
I mean this block appears twice, right?
```
Author 1: Name
Pull request #X merged!
Merge hash: #hash
Would you like to update the associated JIRA? (y/n): y
Enter comma-separated fix version(s) [11.0.0]:
```

Reply:
Ah, they aren't duplicated. One is for Jira and another is for GitHub issue.
We can remove Jjira's one. I've removed it.